syntax = "proto3";

package qtprotobufnamespace.tests;

message SimpleStringMessage {
    string testFieldString = 6;
}

message SimpleIntMessage {
    sint32 testField= 1;
}

message BlobMessage {
    bytes testBytes = 1;
}
